src/omv/ports/stm32/sensor.c

```diff
@@ -452,8 +472,8 @@ int sensor_init()
             return -3;
             break;
     }
+    switch (sensor.chip_id_w) {
```

Comment:
You forgot to add the initialization part here after GC sensor

Reply:
My bad.

Reply:
Wow.. It is stranger...
In my original commit, it is here ... but .. my driver init code is also gone..

https://github.com/openmv/openmv/blob/4f67bdb30bf79776193d6d82b544c1a5a17e0798/src/omv/ports/stm32/sensor.c